### PR TITLE
[WIP] prepare for Rails 5 release

### PR DESCRIPTION
**Pull request by smartinez87**
This PR prepares us to support Rails 5 once it's released.

Tasks
- [x] remove`.beta1` from dependencies

### CODE REVIEW

File: exception_notification.gemspec [hunk 1]
```diff
@@ -1,14 +1,14 @@
 Gem::Specification.new do |s|
   s.name = 'exception_notification'
-  s.version = '4.1.2'
+  s.version = '5.0.0'
   s.authors = ["Jamis Buck", "Josh Peek"]
-  s.date = %q{2015-12-04}
+  s.date = %q{2015-12-28}
```
Comment (spastorino) [inline exception_notification.gemspec]: You don't need to set this http://www.rubydoc.info/github/rubygems/rubygems/Gem%2FSpecification%3Adate%3D

**Comment by clemensg** [comment]
Hi @smartinez87,

because you locked rails to '5.0.0.beta1' in your gemspec, the security fix from today can't be applied (rails beta was bumped to 5.0.0.beta1.1: http://weblog.rubyonrails.org/2016/1/25/Rails-5-0-0-beta1-1-4-2-5-1-4-1-14-1-3-2-22-1-and-rails-html-sanitizer-1-0-3-have-been-released/).

Could you please change the rails dependencies in the gemspec of the rails5 branch to: `gem.add_dependency "foo", ">= 5", "< 6" or something similar?

Thanks.

**Comment by smartinez87** [comment]
@clemensg you should be able to use the latest beta now

**Comment by clemensg** [comment]
Thank you, @smartinez87 !

**Comment by doits** [comment]
You can also use dependency on rails with `~> 5.x` which does include beta and official versions afaik.

**Comment by aried3r** [comment]
Will support for Rails 4 actually be dropped as the README in this PR suggests? I don't see any breaking changes?

**Comment by doits** [comment]
Any chance to release this as beta version compatible with Rails 5 beta?

**Comment by bogdanRada** [comment]
any update on this? Will support for Rails 4 actually be dropped as the README in this PR suggests?  when can be expect a new rubygems release? it has been almost 2 months since last comment on this thread.

**Comment by koic** [comment]
[Rails 5.0.0 has been released](http://weblog.rubyonrails.org/2016/6/30/Rails-5-0-final) 🎉 
How is the progress on this?

**Comment by aried3r** [comment]
I think if we bump Ruby 2.2 to 2.2.2 on Travis, most of the issues on Travis-CI (wrt to Rails 5) should be resolved. For the versions <2.2.2, we need to explicitly require rack <2, I think.

**Comment by alpaca-tc** [comment]
👍  Great fix.

**Pull request merged**